tools/dashboard/generate_dashboard.py

```diff
@@ -109,46 +194,47 @@ def get_sortkey(s):
 
         # find latest commit that should have been tested by now
         threshold = now - timedelta(hours=timeout)
-        ages_beyond_threshold = [i for i, c in enumerate(log) if c["date"] < threshold]
-        threshold_age = ages_beyond_threshold[0]
+        beyond_threshold = [i for i, c in enumerate(log.commits) if c.date < threshold]
+        threshold_age = beyond_threshold[0]
```

Comment:
```python
threshold_age =  next(i for i, c in enumerate(log.commits) if c.date < threshold)
```
... unless I overlooked that the full `beyond_threshold` is also needed somewhere.

Reply:
Neat :-)